.github/workflows/packages.yaml

```diff
@@ -40,55 +32,45 @@ jobs:
         with:
           name: dist
           path: dist
+
+  build-conda:
+    name: Build Conda Package
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
       - uses: conda-incubator/setup-miniconda@v2
         with:
           auto-update-conda: true
       - name: Generate package for conda
         id: conda_build
         run: |
-          conda update conda
+          echo "conda pkgs dir $CONDA_PKGS_DIRS"
           conda install -c conda-forge mamba
-          mamba install -c conda-forge conda-build boa
-          conda mambabuild . -c defaults -c conda-forge -c numba -c rapidsai --output-folder ./conda_packages
-          export CONDA_PACKAGE=$(find ./conda_packages/ -name merlin-core*.tar.bz2)
-          echo "conda_package : $CONDA_PACKAGE"
-          echo "conda_package=$CONDA_PACKAGE" >> $GITHUB_OUTPUT
+          mamba install -c conda-forge conda-build boa conda-verify
+          conda mambabuild . -c defaults -c conda-forge -c numba -c rapidsai -c nvidia --output-folder ./conda_packages
+          conda_package=$(find ./conda_packages/ -name "*.tar.bz2")
+          export CONDA_PACKAGE=$conda_package
+          echo "conda_package : $conda_package"
+          echo "conda_package=$conda_package" >> "$GITHUB_OUTPUT"
       - name: Upload conda artifacts to github
         uses: actions/upload-artifact@v3
         with:
           name: conda
           path: ${{ steps.conda_build.outputs.conda_package }}
 
-      # Build docs, treat warnings as errors
-      - name: Building docs
```

Comment:
This Building docs job has recently started failing with Python 3.8. Removing this, since it doesn't seem relevant to the package build/release

Reply:
I think it may have been included here to make sure we don't release when the docs build is broken?

Reply:
I've added that back as a required job that runs before the package build/release jobs. And removed the dependency  between this Packages workflow and the PR docs Preview Workflow by moving the docs build job with the artifact upload closer to where it's used.

Reply:
To get the Docs PR Preview running as an independent workflow (not triggered by the successful completion of another workflow requires some further work on that re-usable workflow). 

I updated this PR to be only related to the splitting of the jobs into PyPI/Conda build/release only since that is an independent issue